lib/util.js

```diff
@@ -151,7 +151,8 @@ exports.debuglog = function(set) {
     debugEnviron = process.env.NODE_DEBUG || '';
   set = set.toUpperCase();
   if (!debugs[set]) {
-    if (new RegExp(`\\b${set}\\b`, 'i').test(debugEnviron)) {
+    var sets = debugEnviron.split(',').map((s) => s.toUpperCase());
```

Comment:
Maybe we should place all the `process.env.NODE_DEBUG` processing in the `if` clause?